### ISSUE TITLE
Do not destroy Mastodon

### ISSUE THREAD

**Issue by stemy2** [mastodon/mastodon#24652]
### Pitch

Hello,

Some directions taken by Mastodon main dev are pretty worrying because it could make it appealing for a buyout. Here is a thread wich explain everything: https://mstdn.social/@feditips/110260432218416976

I think Eugen Rochko should urgently rethink his strategies.

### Motivation

If Mastodon is bought by a shady multinational (wich is for me a pleonasm), it will be the end of the benefits of the entire fediverse.

**Comment by koyuawsmbrtn** [comment]
I am also a bit worried about Mastodon's future. Sadly not every other server implementation works as well as Mastodon does.

**Comment by Shanesan** [comment]
I don't think this is the place to discuss specific servers or apps. Contribute to feditips' thread.

**Comment by Cassolotl** [comment]
The issue is a button on the app's registration UI, right? So this should be on the app's Github issue list?

**Comment by Shanesan** [comment]
Correct: https://github.com/mastodon/mastodon-ios/issues

**Comment by stemy2** [comment]
If you read the concerned thread, you'll see that the app is not the only problem.

**Comment by Shanesan** [comment]
I've read the concern thread to its fullest, contributed, and agree with the majority of it. But Mastodon the social network is designed to house the Mastodon drama and it's on the server _that is run by the Mastodon founder_ so to imagine he didn't see it yet is silly. 

The issues page isn't for political or economic protests or concerns. There's plenty of technical issues here that need to be solved and we don't need this place to turn into meta issues.

**Comment by Cassolotl** [comment]
> If you read the concerned thread, you'll see that the app is not the only problem.

Then you need to be specific. This is an issue list, so you have to make one issue per specific concern.
